--- ../libsysio-iofwd-update/Rules.make	2009-08-19 14:30:04.000000000 -0500
+++ ./Rules.make	2009-08-10 22:13:19.000000000 -0500
@@ -9,7 +9,11 @@
 STDEV_DEV_CPPFLAGS =
 endif
 
-DEV_CPPFLAGS = $(STDFD_DEV_CPPFLAGS) $(STDDEV_DEV_CPPFLAGS)
+IOFWD_DEV_CPPFLAGS =-DIOFWD_DEV=1 -I$(top_srcdir)/dev/iofwd -I$(top_srcdir)/misc -I/home/copej/mac-copej/work/anl/mcs/iofwd-64bit/iofwd/src/zoidfs -I/home/copej/mac-copej/work/anl/mcs/iofwd-64bit/iofwd
+IOFWD_DEV_LDFLAGS = -L/home/copej/mac-copej/work/anl/mcs/iofwd-64bit/iofwd/src/zoidfs -lzoidfsclient
+
+DEV_CPPFLAGS = $(STDFD_DEV_CPPFLAGS) $(STDDEV_DEV_CPPFLAGS) $(IOFWD_DEV_CPPFLAGS)
+DEV_LDFLAGS = $(IOFWD_DEV_LDFLAGS)
 
 if WITH_THREAD_MODEL_POSIX
 THREAD_MODEL_POSIX_COMPILER_FLAGS=-pthread
@@ -23,3 +27,5 @@
 	$(AUTOMOUNT) $(ZERO_SUM_MEMORY) $(DEV_CPPFLAGS) \
 	$(DEFER_INIT_CWD) $(SYSIO_LABEL_NAMES) $(_HAVE_STATVFS) \
 	-I$(top_srcdir)/include 
+
+AM_LDFLAGS = $(IOFWD_DEV_LDFLAGS)
