--- ../../../iofwd/tmp/iofwd-vanilla/src/iofwd/frontend/IOFWDReadLinkRequest.cpp	2009-07-29 17:37:29.000000000 -0500
+++ ./iofwd/frontend/IOFWDReadLinkRequest.cpp	2009-07-29 17:04:54.000000000 -0500
@@ -11,6 +11,13 @@
 {
    process (req_reader_, handle_);
    process (req_reader_, buffer_length_);
+
+   /*
+    * Init param_ with the decoded XDR data
+    */
+   param_.buffer_length = buffer_length_;
+   param_.handle = &handle_;
+
    return param_;
 }
 
